### ISSUE TITLE
Jetpack Settings Page: Toggles can be selected from whitespace in toggle span element

### ISSUE THREAD

**Issue by chickenn00dle** [Automattic/jetpack#13351]
#### Steps to reproduce the issue

1. Go to the Jetpack Settings page in wp-admin
2. Click on the whitespace next to any of the toggle descriptions
3. This will activate the toggle

#### What I expected

Clicking on the whitespace should NOT lead to a toggle activation. This can be tricky for users and lead to accidentally turning on/off a feature.

#### What happened instead
The toggle is activated despite not having actually selecting it.

**Screenshots**
Screen Recording Link: https://d.pr/i/QUXA51

**Comment by jeherve** [comment]
I actually like that, it makes it easier to turn things on and off, but I can see how it could be confusing for some.

I'd like to loop in Design on this to get their opinion and their help if we need to make any changes there.

**Comment by jeherve** [comment]
Related: https://github.com/Automattic/wp-calypso/issues/10949

**Comment by folletto** [comment]
I'd clarify that the related topic is about the _label_ being clickable, not the whitespace _after_ the label, which I agree it's confusing.

I also did a quick check and in Calypso the issue is there too, even if not always: "Infinite Scroll" and "Feed Settings" have clickable whitespace, but "Content Types" don't. 

![Screenshot 2019-08-28 at 10 32 33](https://user-images.githubusercontent.com/4389/63843977-38a6dd00-c97f-11e9-8039-5da40e82c001.png)


So, to be clear:

* The label should be clickable
* The whitespace _after_ the label should not

**Comment by scottsweb** [comment]
This is due to our form inputs being wrapped by the `label` and that label is set to `display:block`. There is another issue open to look at this here: https://github.com/Automattic/jetpack/issues/13755

It can be fixed for the most part with `display: table` ... but that does seem to impact layout on other input groups. 

My preference is to close this issue and focus on updating the markup in #13755
